MIDAS/src/hardware/Camera.cpp

```diff
@@ -0,0 +1,60 @@
+#include <ACAN2517.h>
+#include <SPI.h>
+
+static const byte MCP2517_SCK  = 0;
+static const byte MCP2517_MOSI = 0;
+static const byte MCP2517_MISO = 0;
+
+static const byte MCP2517_CS  = 0;
+static const byte MCP2517_INT = 0;
+static bool debug = false;
+
+ACAN2517 can (MCP2517_CS, SPI, MCP2517_INT) ;
+
+void setup () {
+
+  Serial.begin(9600);
+
+  while (!Serial) {
+    delay (50) ;
+  }
+
+  SPI.begin (MCP2517_SCK, MCP2517_MISO, MCP2517_MOSI) ;
```

Comment:
spi is begun globally in systems.cpp or main.cpp i forgot which, but you don't need to worry about beginning spi